docs/_docs/installation.md

```diff
@@ -50,6 +82,16 @@ gem install jekyll
 All of Jekyll’s gem dependencies are automatically installed by the above
 command, so you won’t have to worry about them at all.
 
+<div class="note info">
+  <h5>Permissions Issues?</h5>
+  <p>
+    The above command performs a system-wide install of jekyll (on most
+    systems). If you get a permissions error, you can either run the command as root
+    (<code>sudo gem install jekyll</code>) or use
```

Comment:
This always leads to conflicts with multiple versions of Jekyll being installed. There have been countless bug reports from users who installed Jekyll using `sudo` without understanding what that meant. Let’s recommend installing to `~/` instead of using `sudo`

Reply:
Good point. No sudo.